qa/workunits/ceph-disk/ceph-disk.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
-if [ -f $(dirname $0)/../../standalone/ceph-helpers-root.sh ]; then
-    source $(dirname $0)/../../standalone/ceph-helpers-root.sh
+if [ -f $(dirname $0)/../../standalone/ceph-helpers.sh ]; then
+    source $(dirname $0)/../../standalone/ceph-helpers.sh
```

Comment:
shouldn't this switch back to ../../workunits/ceph-helpers-root.sh?  (I'm guessing the -root version is different?)